models/source/im-bm-ci/im_msrtd_format.sql

```diff
@@ -0,0 +1,20 @@
+{{
+  config({
+    "materialized" : "table",
+    "post-hook" : [
+      "grant select on table {{this}} to group non_gl_read_only"
+      ]
+    })
+}}
+
+select
+    udf_msrtd_format_id::int msrtd_format_id,
+    udf_format_desc as format_desc,
+    nullif(udf_sub_unit_format, '')::int as sub_unit_format,
```

Comment:
This is an id, and should be specified as such in the name. Can you rename this to sub_unit_format_id?